gateway/README.md

```diff
@@ -2,116 +2,185 @@
-curl -i https://hello.kyma.local
-# for the service
-curl -ik https://http-db-service.ykyma.local/orders
+curl -ik https://hello.kyma.local
+# > 200 Hello world
 ```
 
-#### Test the APIs with authentication  
+#### Expose service with authentication
 
-Run the following commands in either the 'lambda' or the 'service' folder.
+``` bash
+# Create Api with default authentication settings
+kubectl apply -f ./lambda/api-with-default-auth.yaml -n <namespace>
 
-```bash
-kubectl apply -f api-with-auth.yaml -n $KYMA_EXAMPLE_ENV
+# OR
+
+# Create Api with custom authentication settings
+kubectl apply -f ./lambda/api-with-auth.yaml -n <namespace>
 ```
 
-Retrieve a token by accessing the [Dex Web application](https://dex-web.kyma.local/). Sign in with the admin@kyma.cx email address and the generic password from the [Dex ConfigMap](../../../resources/core/charts/dex/templates/pre-install-dex-config-map.yaml) file.
+There are two possible ways of exposing secured Api - using default authentication settings and using custom settings. Authentication settings consist of JWKS URI and Issuer. 
 
-Call the secured APIs:
+To use default settings omit or set empty list in authentication field and set:
 
-```bash
-# for the function:
-curl -i https://hello.kyma.local -H 'Authorization: Bearer <insert copied token here>'
-# for the service
-curl -ik https://http-db-service.kyma.local/orders -H 'Authorization: Bearer <insert copied token here>'
+``` yaml
+authenticationEnabled: true
 ```
 
-### Cleanup
+that forces authentication to be enabled, so if authentication settings are not provided it will use default ones.
+
+In api-with-default-auth.yaml "authentication" field is omitted.
+
+To use custom settings fill "authentication" field like it is done in api-with-auth.yaml sample. You can omit "authenticationEnabled" field or set its value to true.
 
-Run the following command to completely remove the example and all its resources from the cluster:
+#### Test the APIs with authentication
 
 ```bash
-# for the function:
-kubectl delete function,api -l example=serverless-lambda -n $KYMA_EXAMPLE_ENV
+# without token:
+curl -ik https://<hostname>.kyma.local
+# > 401 Origin authentication failed.
 
-# for the service:
-kubectl delete service,deployment,api -l example=http-db-service -n $KYMA_EXAMPLE_ENV
+# with token:
+curl -ik https://<hostname>.kyma.local -H 'Authorization: <token>'
+# > 200 Hello world
 ```
 
-### Troubleshooting
+#### Cleanup
 
-- Exposing an API with authentication takes some time to work. Looking through the API resource can help. A good API should have:
+Remove the Api by using kubectl delete on the latest Api resource you've applied. E.g. if you created Api from api-with-auth.yaml file just do:
 
-```yaml
-authenticationstatus
-  code: 2
+```bash
+kubectl delete -f ./lambda/api-with-auth.yaml -n <namespace>
 ```
 
+Remove service or lambda by simply using kubectl delete on the resource from which the resource was created. E.g. for lambda:
```

Comment:
``Remove a service or lambda by using the `kubectl delete` command` on the resource from which the resource was created. See the example for lambda:``

Reply:
Should actually be "[...] on the file [...]" so I changed that.